Remove wwan panel on Solaris since we don't have libnm and libnma.

Not suitable for upstream.

--- a/panels/meson.build	2022-02-28 23:28:06.078834194 -0800
+++ b/panels/meson.build	2022-02-28 23:28:16.724239203 -0800
@@ -24,7 +24,6 @@
   'sound',
   'universal-access',
   'usage',
-  'wwan',
 ]
 
 if host_is_linux
